bowling-ball/Frames.cs

```diff
@@ -0,0 +1,64 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BowlingBall
+{
+    class Frames : IFrames
+    {
+        int[] EachFrame = new int[2];

```

Comment:
does a frame has a frame? (`Frames` having `EachFrame` is confusing)
use better naming convention real to the game